### ISSUE TITLE
Failed to install 0.1.4

### ISSUE THREAD

**Issue by kpp** [nrc/cargo-src#248]
`cargo +nightly install cargo-src`:

```
error[E0432]: unresolved import `syntax::codemap`                               
  --> /home/humbug/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-src-0.1.4/src/highlight.rs:22:13
   |                                                                            
22 | use syntax::codemap::{CodeMap, FilePathMapping, Loc};                      
   |             ^^^^^^^ Could not find `codemap` in `syntax`                   
                                                                                
error[E0599]: no method named `codemap` found for type `syntax::parse::ParseSess` in the current scope
  --> /home/humbug/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-src-0.1.4/src/highlight.rs:40:23
   |                                                                            
40 |         let fm = sess.codemap().new_filemap(FileName::Real(PathBuf::from(&file_name)), file_text);
   |                       ^^^^^^^                                              
                                                                                
error[E0599]: no method named `codemap` found for type `syntax::parse::ParseSess` in the current scope
  --> /home/humbug/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-src-0.1.4/src/highlight.rs:42:69
   |                                                                            
42 |         let mut out = Highlighter::new(analysis, project_path, sess.codemap());
   |                                                                     ^^^^^^^
                                                                                
error[E0599]: no method named `codemap` found for type `syntax::parse::ParseSess` in the current scope
  --> /home/humbug/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-src-0.1.4/src/highlight.rs:46:94
   |                                                                            
46 |         let mut classifier = Classifier::new(lexer::StringReader::new(&sess, fm, None), sess.codemap());
   |                                                                                              ^^^^^^^
                                                                                
error: aborting due to 4 previous errors                                        
                                                                                
Some errors occurred: E0432, E0599.                                             
For more information about an error, try `rustc --explain E0432`.               
error: failed to compile `cargo-src v0.1.4`, intermediate artifacts can be found at `/tmp/cargo-install4HOqT7`

Caused by:
  Could not compile `cargo-src`.

To learn more, run the command again with --verbose.
```

```
$ rustc -V
rustc 1.30.0-nightly (2ab3eba30 2018-09-14)
```

Whereas I managed to install in from git:

```
cargo +nightly install --git https://github.com/nrc/cargo-src.git
```

**Comment by nrc** [comment]
Published 0.1.5 which should work.

**Comment by kpp** [comment]
Fixed.
